app/Models/Model.php

```diff
@@ -77,6 +77,17 @@ public function refresh()
         return parent::refresh();
     }
 
+    public function scopeCursorSort($query, array $sort, ?array $cursor)
+    {
+        if (empty($cursor)) {
+            foreach ($sort as $sortItem) {
+                $query->orderBy($sortItem['column'], $sortItem['order']);
+            }
+        } else {
+            $query->cursorWhere($cursor);
+        }
+    }
```

Comment:
this part seems weird since `$sort` gets ignored if `$cursor` is set but `$cursor` always seems to be `$sort` + extra `value` key when not `null`, making it pretty much `scopeCursorWhere` where the `where` part is skipped if there's no value? 🤔 ❓